spring-xd-dirt/src/test/java/org/springframework/xd/dirt/integration/kafka/KafkaMessageBusTests.java

```diff
@@ -16,53 +16,44 @@
 
 package org.springframework.xd.dirt.integration.kafka;
 
-import static org.junit.Assert.assertArrayEquals;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
+import static org.junit.Assert.*;
```

Comment:
DITO. Although I see that SF uses this setting, but I don't think that it is good for some methods ambiguity from different classes